manifest.toml

```diff
@@ -32,24 +33,19 @@ ram.runtime = "50M"
     [install.domain]
     type = "domain"
 
-    [install.path]
-    type = "path"
-    default = "/aeneria"
```

Comment:
Weird, we can still change the path after the upgrade.

Reply:
yep, I saw that too, but if you try to put a path, I think it fails